doc/source/whatsnew/v0.15.0.rst

```diff
@@ -255,12 +252,12 @@ Additionally :meth:`~pandas.DataFrame.memory_usage` is an available method for a
 ^^^^^^^^^^^^
 
 ``Series`` has gained an accessor to succinctly return datetime like properties for the *values* of the Series, if its a datetime/period like Series. (:issue:`7207`)
-This will return a Series, indexed like the existing Series. See the :ref:`docs <basics.dt_accessors>`
 
+This will return a Series, indexed like the existing Series. See the :ref:`docs <basics.dt_accessors>`
 .. ipython:: python
```

Comment:
Can you revert this change? (there needs to be a blank line between the last sentence and `.. ipython::`)